### ISSUE TITLE
Warning for missing annotation source code

### ISSUE THREAD

**Issue by GoogleCodeExporter** [anne/google-gin#182]
```
What steps will reproduce the problem?
1. Use an annotation in GWT client code and don't include the source for that 
annotation. In my particular case I'm using javax.annotation.Nullable, where 
the class file is included as a Guava dependency.

What is the expected output? What do you see instead?
Expected: No warning is printed since we can't make use of annotations at GWT 
runtime anyway (am I wrong about this?)

Result: Warning is printed for missing annotation source code


What version of the product are you using? On what operating system?
GIN 2.0 / Ubuntu 12.04

Please provide any additional information below.

```

Original issue reported on code.google.com by `mcart...@gmail.com` on 24 Sep 2012 at 3:48
- Merged into: #166

**Comment by GoogleCodeExporter** [comment]
```
Just got fixed in HEAD. :)
```

Original comment by `aragos` on 25 Sep 2012 at 2:34
- Changed state: **Duplicate**

**Comment by GoogleCodeExporter** [comment]
```
I still see this error in gin 2.0
```

Original comment by `david.nouls` on 4 Apr 2013 at 11:38

**Comment by GoogleCodeExporter** [comment]
```
Aha the fix happened a few days after the 2.0 release, too bad :-)
```

Original comment by `david.nouls` on 4 Apr 2013 at 11:41
